### PR TITLE
[#1416] improvement(docs): Improve Glossary

### PR DESCRIPTION
**Pull request by hiirrxnn**
What changes were proposed in this pull request?
Improvement of glossary

Why are the changes needed?
So that users can resolve their doubts regarding a term easily and more precisely.

Fix: #1416 

Does this PR introduce any user-facing change?
N/A.

How was this patch tested?
N/A.

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -85,6 +101,10 @@ This software is licensed under the Apache License version 2."
 
 - Variables used to pass information to running processes.
 
+## Geo-Distributed
+
+- Refers to the distribution of data or services across multiple geographic locations.
```
[the remark this answers is not in the document]
Comment (hiirrxnn) [reply docs/glossary.md]: Done !

File: docs/glossary.md [hunk 2]
```diff
@@ -177,18 +201,38 @@ This software is licensed under the Apache License version 2."
...
+
+- A feature introduced in Gradle to detect and manage JDK versions. 
+
 ## JDBC URI
 
 - The JDBC connection address specified in the catalog configuration, including details such as the database type, host, port, and database name.
 
+## JMX 
+
+- Java Management Extensions, a Java technology that provides tools for managing and monitoring Java applications.
```
[the remark this answers is not in the document]
Comment (hiirrxnn) [reply docs/glossary.md]: Done!

**Comment by justinmclean** [comment]
What you have added is a good start, but I think we can add more terms.

**Comment by jerryshao** [comment]
@hiirrxnn Do you have any further modifications? I'm OK with the current change, we can always add new glossaries later on.

**Comment by hiirrxnn** [comment]
You can merge this right now , I will add more glossary once I'm done with the other issues I'm working on in this repo !

**Comment by jerryshao** [comment]
Can you please update the PR description to follow our template?

**Comment by hiirrxnn** [comment]
Updated!

**Review by jerryshao — APPROVED** [review]

**Pull request merged**